pom.xml

```diff
@@ -63,7 +63,7 @@
     <gwteventservice.version>1.2.1</gwteventservice.version>
     <okapi.version>0.22</okapi.version>
 
-    <zanata.assets.version>7.4-SNAPSHOT</zanata.assets.version>
+    <zanata.assets.version>8.0-SNAPSHOT</zanata.assets.version>
```

Comment:
@lukebrooker updated something in that to support the new UI, so we'll want his input into what we can do here.